public/dorequest.php

```diff
@@ -388,6 +448,56 @@ function DoRequestError(string $error, ?int $status = 200, ?string $code = null)
         }
         break;
 
+    case "syncachievements":
```

Comment:
This implies the achievement definitions will be synced with some external source. Recommend "awardachievements". "syncunlocks" would also be acceptable, but still implies syncing entire records, not generating new ones.

Reply:
Renamed to "awardachievements".